### PR TITLE
hubspot form refactor and adding new forms

### PR DESCRIPTION
**Pull request by calellowitz**
@proteusvacuum 
small refactor of the way we send forms to hubspot to try to clean it up a bit, and I also added a few new forms.

### CODE REVIEW

File: corehq/apps/analytics/tasks.py [hunk 1]
```diff
@@ -283,13 +285,10 @@ def track_confirmed_account_on_hubspot(webuser):
...
-def track_entered_form_builder_on_hubspot(webuser, cookies, meta):
-    _send_form_to_hubspot(HUBSPOT_FORM_BUILDER_FORM_ID, webuser, cookies, meta)
-
-
-@analytics_task()
-def track_app_from_template_on_hubspot(webuser, cookies, meta):
-    _send_form_to_hubspot(HUBSPOT_APP_TEMPLATE_FORM_ID, webuser, cookies, meta)
+def send_hubspot_form(form_id, request):
+    if request and request.couch_user.is_web_user():
+        meta = get_meta(request)
```
Comment (proteusvacuum) [inline corehq/apps/analytics/tasks.py]: this might not work since I don't think a `request` object is picklable.

**Comment by calellowitz** [comment]
@proteusvacuum 
good catch. i fixed that task to not need the request. let me know if it looks alright to you

**Comment by proteusvacuum** [comment]
Looks fine, but lots of test failures and submodule updates (were those intentional?)

**Comment by calellowitz** [comment]
@proteusvacuum fixed test failures

**Review by proteusvacuum — APPROVED** [review]

**Pull request merged**